Getting Started (#148)
MicrosoftDocs/OData-docs: Odata-docs/client/v7/getting-started.md

In this article we are going to focus on the latter approach.

## What are Code Generation tools

> * A tool used to generate a client proxy file for an Odata service. It used the schema to generate the proxy classes.

Reviewer: ```suggestion
* These are tools used to generate client proxy file for an Odata service. They use the schema to generate the proxy classes.
```
